nuxeo-services/nuxeo-platform-dublincore/src/main/java/org/nuxeo/ecm/platform/dublincore/NXDublinCore.java

```diff
@@ -37,9 +37,11 @@ private NXDublinCore() {
 
     /**
      * Locates the core service using NXRuntime.
+     * 
+     * @Deprecated {@link Framework#getService(Class)}
      */
-    public static DublinCoreStorageService getDublinCoreStorageService() {
-        return (DublinCoreStorageService) Framework.getRuntime().getComponent(DublinCoreStorageService.ID);
+    public static DublinCoreStorageServiceImpl getDublinCoreStorageService() {
```

Comment:
Good point to introduce DublinCoreStorage interface, But you shouldn't change this method signature.
As we now have an interface, you can deprecate this method by annotating it with `@Deprecated` and by adding `@deprecated since 10.2, use directly {@link Framework#getService(Class)} instead` and remove all usage in our code.